### DISCUSSION TITLE
How can I hide context menu for non-admin users

### DISCUSSION THREAD

**Discussion by saqibvohra93** [directus/directus#16713] [Archive]
I need to hide the context menu which allows to Show/Hide Hidden Collections in the content navigation pane.

I have located the code, but not sure how to put conditions to only display it for admin users. 

https://github.com/saqibvohra93/directus-a/blob/86dd593f91ea531ce7fec5ad147744dbafea3d4d/app/src/modules/content/components/navigation.vue#L25

Would really appreciate it if someone can kindly help with this.

Thanks

**Comment by rijkvanzanten** [comment]
There's currently no setting or extension that allows for disabling that.

**Reply by saqibvohra93** [reply]
Thanks @rijkvanzanten. Looking forward to being able to do more configs from within DirectUs.
